### DISCUSSION TITLE
Search options, not possible?

### DISCUSSION THREAD

**Discussion by HubKing** [pystardust/ytfzf#640] [Q&A]
I mean things like below. Also, it seems that it only shows the first page. Can I get more results (analogous to scrolling down on actual YouTube search result page)?

![image](https://user-images.githubusercontent.com/10900489/216733047-8f9ad01d-a90a-414f-b9da-4f071f9c462f.png)

**Comment by Euro20179** [comment]
For the search options check the manual, it's in `ytfzf(1)`, the shortcuts are also listed in the manual, `ctrl-p` can be pressed to see more results
